h5_test/tst_h_atts3.c

```diff
@@ -46,7 +46,11 @@ main()
       hid_t file_typeid1[NUM_OBJ], native_typeid1[NUM_OBJ];
       hid_t file_typeid2, native_typeid2;
       hsize_t num_obj;
+#if H5_VERSION_GE(1,12,0)
```

Comment:
Unrelated to this PR, but a note. @DennisHeimbigner, this H5_VERSION_GE macro seems useful, I wonder if we should add a netcdf equivalent? We can discuss later, mentioning it now before I forget (again).

Reply:
You mean something like NC_VERSION_GE?